templates/suitecrm/meta.yaml

```diff
@@ -0,0 +1,35 @@
+name: SuiteCRM
+description: SuiteCRM is a completely open source, enterprise-grade Customer Relationship Management (CRM) application.
+instructions: null
+changeLog:
+  - date: 2023-1-12
+    description: first release
+links:
+  - label: Github
+    url: https://github.com/salesagility/SuiteCRM
```

Comment:
Can you please add the website? https://suitecrm.com/. You can add the docs as well: https://docs.suitecrm.com/